00_make_block_group_shp.R

```diff
@@ -44,8 +48,14 @@ saveRDS(tracts_sf_1980, "tracts_1980_5072.rds")
 # 1970
 tracts_sf_1970 <- st_read('/Users/RASV5G/Downloads/nhgis0018_shape/nhgis0018_shapefile_tl2000_us_tract_1970/US_tract_1970.shp')
 
+summary(as.numeric(tracts_sf_1970$NHGISST))
+
```

Comment:
this doesn't need to be in there, right?

Reply:
nope, I accidentally left that in. I will remove it.